GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -289,6 +297,13 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FormBrowse.Command.StashPop, Keys.Control | Keys.Alt | Keys.Down),
                     Hk(FormBrowse.Command.ToggleBetweenArtificialAndHeadCommits, Keys.Control | Keys.OemBackslash),
                     Hk(FormBrowse.Command.ToggleBranchTreePanel, Keys.Control | Keys.Alt | Keys.C)),
+                new HotkeySettings(
+                    RepoObjectsTree.HotkeySettingsName,
+                    Hk(RepoObjectsTree.Command.Delete, Keys.Delete),
+                    Hk(RepoObjectsTree.Command.Rename, Keys.F2),
+                    Hk(RepoObjectsTree.Command.Search, Keys.F3),
+                    Hk(RepoObjectsTree.Command.MultiSelect, Keys.Control | Keys.Space),
```

Comment:
This overrides the current (global)shortcut, opening FormCommit
No good suggestion right now...

Reply:
Only the Left Panel is affected. Overriding the hotkey `Ctrl+Space` there is intentional because it is the standard hotkey for manipulating multi-selections using the keyboard. 
https://github.com/gitextensions/gitextensions/pull/9723#issuecomment-1064817977

Reply:
It could be seen as a regression for users that do not change focus when submitting
No better suggestion, approving